src/Behat/Testwork/Call/CallResults.php

```diff
@@ -18,6 +18,8 @@
  * Aggregates multiple call results into a collection and provides an informational API on top of that.
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ *
```

Comment:
The `IteratorAggregate` interface is defined as a template, so we need to specify the type here when implementing it